src/Worker.Extensions.DurableTask/ITokenSource.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask
+{
+    /// <summary>
+    /// Implementations of this interface can be used to provide authorization tokens for outbound HTTP requests.
+    /// This calss is the same as the one in WebJobs.Extensions.DurableTask/ITokenSource.cs
+    /// The implementation is kept in sync to ensure compatibility.
+    /// </summary>
+    public interface ITokenSource
```

Comment:
Including this as a public contract is misleading. No custom implementation of this will ever be usable on the WebJobs side (which is where it is actually used), because whatever assembly that implements it will not be loaded there.

This was a limitation for out of proc with the current HTTP polling approach.